test/e2e/network_test.go

```diff
@@ -59,6 +72,181 @@ const (
 	powerShellExe                 = "pwsh.exe"
 )
 
+// startPacketTrace starts a packetmon packet trace on the given Windows node
+func (tc *testContext) startPacketTrace(node *v1.Node) error {
+	addr, err := controllers.GetAddress(node.Status.Addresses)
+	if err != nil {
+		return err
+	}
+	cmd := "mkdir C:\\debug; pktmon start --file-name C:\\debug\\pktmon.etl --capture --trace -p 0c885e0d-6eb6-476c-a048-2457eed3a5c1 -p Microsoft-Windows-TCPIP -l 5 " +
```

Comment:
It may be hard to track/maintain what provider the GUIDs are referring to.  Please consider defining a constant explicitly naming them e.g. `$WindowsNetworkingAdapter=0c885e0d-6eb6-476c-a048-2457eed3a5c1`